### PR TITLE
Fix: press release edits

### LINKED ISSUE #249: Press Release: 2 edits needed
Hello,

Two edits are needed in [this press release](https://www.calitp.org/press/four-northern-california-transit-agencies-join-forces-to-buy-contactless-open-loop-fare-payment-systems-off-of-californias-purchasing-agreements).

- [ ] Paragraph 2, Sentence 1:
Change _from:_ 
Humboldt Transit Authority, Lake Transit Authority, Mendocino Transit Authority, and Redwood Coast Transit Authority—collectively part of the Far North Group—are the first group ...

_Change to:_
Humboldt Transit Authority, Lake Transit Authority, Mendocino Transit Authority, and Redwood Coast Transit Authority—collectively part of North State Super Region working group, also known as the Far North Group—are the first group ...

- [ ] Paragraph 5, Sentence 1:
Change _from:_ 
No matter which of the four agencies’ fixed-route buses or dial-a-ride vehicles they board, ...

_Change to:_
No matter which of the four agencies’ fixed-route buses they board, ...

### PR DESCRIPTION
**Pull request by angela-tran**
Closes #249

### CODE REVIEW

**Review by thekaveman — APPROVED** [review]
LGTM, let's wait for content approval from @esquared415

**Review by esquared415 — APPROVED** [review]
This looks great. Thanks so much, @angela-tran + @thekaveman! 🥇

**Pull request merged**
